main.yml

```diff
@@ -763,8 +763,12 @@ application-services:
       - id: service-registry
         title: "Service Registry"
         group: overview
-      - id: streams
-        group: overview
+      - id: subscriptions
```

Comment:
These changes are not required. Navigation is not using this config.

Reply:
Reverted.

When would I need to update this location?